packages/xrpl/package.json

```
@@ -46,13 +44,13 @@
     "build:browserTests": "webpack --config ./test/webpack.config.js",
     "analyze": "run-s build:web --analyze",
     "watch": "run-s build:lib --watch",
-    "clean": "rm -rf dist",
+    "clean": "rm -rf dist build coverage",
     "docgen": "typedoc && echo js.xrpl.org >> ../../docs/CNAME",
     "prepublish": "run-s clean build",
-    "test": "nyc mocha --config=test/.mocharc.json --exit",
-    "test:integration": "TS_NODE_PROJECT=tsconfig.build.json nyc mocha ./test/integration/**/*.ts ./test/integration/*.ts",
-    "test:browser": "npm run build:browserTests && TS_NODE_PROJECT=tsconfig.build.json nyc mocha ./test/browser/*.ts",
-    "test:watch": "TS_NODE_PROJECT=src/tsconfig.json mocha --config=test/.mocharc.json --watch --reporter dot",
+    "test": "jest --verbose false --silent=false --runInBand ./test/*.test.ts ./test/models/*.test.ts ./test/utils/*.test.ts ./test/wallet/*.test.ts",
```

I think this should also include `./test/client/*.test.ts`. This also seems a bit brittle to new test folders being created. Maybe it'd be better to specify it by excluding `integration` and `fixtures` instead of including unit tests explicitly:
```suggestion
    "test": "jest --verbose false --silent=false --runInBand ./test/**/*.test.ts --testPathIgnorePatterns=(./test/integration|./test/fixtures)",
```
https://jestjs.io/docs/cli#--testpathignorepatternsregexarray